Examples/ObjectiveC/ClusteringWithImagesExample.m

```diff
@@ -0,0 +1,69 @@
+//
+//  ClusteringWithImagesExample.m
+//  Examples
+//
+//  Created by Joshua Erb on 7/30/18.
+//  Copyright © 2018 Mapbox. All rights reserved.
+//
+
+#import "ClusteringWithImagesExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleClusteringWithImages = @"ClusteringWithImagesExample";
+
+@interface ClusteringWithImagesExample () <MGLMapViewDelegate>
+
+@property (nonatomic) MGLMapView *mapView;
+@property (nonatomic) UIImage *icon;
+
+@end
+
+@implementation ClusteringWithImagesExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    // Initialize the Map
+    self.mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds
+                                            styleURL:[MGLStyle lightStyleURL]];
+    self.mapView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
+    self.mapView.delegate = self;
+    [self.view addSubview:self.mapView];
+    
+    self.icon = [UIImage imageNamed:@"circle"];
+}
+
+- (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style {
+    NSURL *url = [NSURL fileURLWithPath:[[NSBundle mainBundle] pathForResource:@"ports" ofType:@"geojson"]];
+    
+    // Retrieve data and set as style layer source
+    MGLShapeSource *source = [[MGLShapeSource alloc] initWithIdentifier:@"clusteredPorts" URL:url options:@{
+                              MGLShapeSourceOptionClustered:@(YES),
+                              MGLShapeSourceOptionClusterRadius: @(self.icon.size.width)
+    }];
+    [style addSource:source];
+    
+    MGLSymbolStyleLayer *numbersLayer = [[MGLSymbolStyleLayer alloc] initWithIdentifier:@"clusteredPortsNumbers" source:source];
+    numbersLayer.textColor = [NSExpression expressionForConstantValue:[UIColor whiteColor]];
+    numbersLayer.textFontSize = [NSExpression expressionForConstantValue:@(self.icon.size.width / 2)];
+    numbersLayer.iconAllowsOverlap = [NSExpression expressionForConstantValue:@(YES)];
+    
+    // Style clusters
```

Comment:
The `cloud` image got me confused with the `circle` image (since they're the same color and look similar). Perhaps replacing the cloud with a triangle or some other distinctive shape would help make this more clear?